### ISSUE TITLE
flake8 ignores files when `subproccess.run` called with `args: List`

### ISSUE THREAD

**Issue by rows-s** [discrimy/linthell#3]
When linthell calls `flake8` as `linter_process` all passed files are ignored by `flake8` and it linters all the files in current directory and sub directories. 

To reproduce:

1. in directory with installed `flake8` create two files: 
    - `bad.py` with any error that `flake8` must report (i used `import flake8\n`)
    -  `good.py`, leave it empty so there is no error in it 
2. run
 ```python
 import subprocess
subprocess.run(
    ['flake8', 'good.py'],
    text=True,
    stdout=subprocess.PIPE,
    stderr=subprocess.PIPE,
    shell=True,
    check=False,
)
```

Output:
```python
CompletedProcess(args=['flake8', 'good.py'], returncode=1, stdout="./bad.py:1:1: F401 'flake8' imported but unused\n", stderr='')
```

# Solution

That works fine:

```python
subprocess.run(
    'flake8 good.py',
    text=True,
    stdout=subprocess.PIPE,
    stderr=subprocess.PIPE,
    shell=True,
    check=False,
)
```

Output:
```python
CompletedProcess(args='flake8 good.py', returncode=0, stdout='', stderr='')
```

# env info

flake8=="6.0.0"

**Comment by discrimy** [comment]
Don't reproduce. `linthell`'s function do not include `shell=True`, maybe what's why your example do not work?

```python
>>> from linthell.utils.linters import run_linter_and_get_output
>>> run_linter_and_get_output('flake8', ['good.py'], 'stdout')
''
>>> run_linter_and_get_output('flake8', ['bad.py'], 'stdout')
"bad.py:1:1: F401 'flake8' imported but unused\nbad.py:1:14: W292 no newline at end of file\n"
```

linthell==1.0.2
flake8==6.0.0

Tested on WSL2

**Comment by rows-s** [comment]
Yeah, seem you're being right. I've faced the issue using released version of `linthell` (1.0.1)

**Comment by discrimy** [comment]
`1.0.3` just released! 
https://pypi.org/project/linthell/1.0.3/
